src/main/java/xyz/nucleoid/creator_tools/workspace/editor/ServersideWorkspaceEditor.java

```diff
@@ -72,6 +79,36 @@ public boolean useRegionItem() {
         return true;
     }
 
+    @Override
+    public boolean useRegionVisibilityFilterItem(Collection<String> regions) {
```

Comment:
`@Nullable` parameter? Or alternatively, passing the `Predicate<String>` directly?

Reply:
I changed this method to require a `Predicate<String>` parameter directly in 1fb476ab0232e78158359a7b7fcf9076d4a1c4db.